samples/filler/line.html

```diff
@@ -0,0 +1,151 @@
+<!doctype html>
+<html>
+<head>
+	<title>filler > line | Chart.js example</title>
+	<link rel="stylesheet" type="text/css" href="../style.css">
+	<script src="../../dist/Chart.bundle.js"></script>
+	<script src="../utils.js"></script>
+	<script src="analyser.js"></script>
```

Comment:
I'm not seeing any output in the code section for both samples. Is that expected?